geopandas/plotting.py

```diff
@@ -53,58 +61,52 @@ def plot_polygon_collection(ax, geoms, colors_or_values, plot_values,
 
     geoms : a sequence of `N` Polygons and/or MultiPolygons (can be mixed)
 
-    colors_or_values : a sequence of `N` values or RGBA tuples
-        It should have 1:1 correspondence with the geometries (not their components).
+    values : a sequence of `N` values
+        Values will be mapped to colors using vmin/vmax/cmap. They should
+        have 1:1 correspondence with the geometries (not their components).
+        Otherwise follows `color` / `facecolor` kwargs.
+
+    edgecolor : single color or sequence of `N` colors
+        Color for the edge of the polygons
 
-    plot_values : bool
-        If True, `colors_or_values` is interpreted as a list of values, and will
-        be mapped to colors using vmin/vmax/cmap (which become required).
-        Otherwise `colors_or_values` is interpreted as a list of colors.
+    facecolor : single color or sequence of `N` colors
+        Color to fill the polygons. Cannot be used together with `values`.
```

Comment:
Thinking about it know, I think it can still be informative to include (what we think) the most important ones explicitly in the docstring (face/edgecolor, linewidth), as otherwise you already have to go search for the matplotlib docs (but indeed there are many mores that we certainly shouldn't mention).

For now, `edgecolor` cannot be removed from the signature, because it has a non-default value (but depending on discussion in https://github.com/geopandas/geopandas/issues/318, I will change / remove that)